Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/Climber.java

```diff
@@ -17,5 +20,49 @@ public void initDefaultCommand() {
         // Set the default command for a subsystem here.
         //setDefaultCommand(new MySpecialCommand());
     }
+    
+    
+   
+
+    /**
+	 * 
+	 */
+	public void runClimber() {
+		attachToLift();
+		attachToBar();
+		goUp();
+		
+	}
+	
```

Comment:
I'm a little confused what this method is meant to do. What I would have guessed a runClimber() method would do would just set the climberMotor to the speed parameter, so pretty basic. I see what you're trying to do: the whole climbing process right? Calling all of the individual steps together to make the full process is more of what happens in a Command (i.e. in the execute method), so I would wait for that and not put that here. Maybe just make it a simple runClimber(double speed) method as described above.